src/components/flow/routers/localization/RouterLocalizationForm.tsx

```diff
@@ -120,7 +120,15 @@ export default class RouterLocalizationForm extends React.Component<
 
       const { verboseName } = getOperatorConfig(originalCase.type);
 
-      if (originalCase.arguments && originalCase.type !== Operators.has_number_between) {
+      const numericOperators = Object.values(Operators).filter(operator => {
+        return (
+          operator.startsWith(Operators.has_date) ||
+          operator.startsWith(Operators.has_time) ||
+          operator.startsWith(Operators.has_number)
```

Comment:
this feels like a little weird using `Operators.has_date` which means the `has_date` rule.. to mean the prefix `"has_date"` but I guess it works...